src/types/backEnd/PollutedLocationResponse.ts

```diff
@@ -35,5 +39,9 @@ export const mapToPollutedLocation: (
           sensitivity: "accent",
         }) === 0
     ),
+    events:
+      from.events == undefined
+        ? undefined
+        : from.events.map(mapToCleaningEvent),
```

Comment:
Can't this be written as `events: from.events?.map(mapToCleaningEvent)` ?

Reply:
true, good point.
knew js had bot of the equality operators, never really checked out how do they differ. nice